packages/rest-typings/src/apps/index.ts

```diff
@@ -5,16 +5,21 @@ import type { IPermission } from '@rocket.chat/apps-engine/definition/permission
 import type { ISetting } from '@rocket.chat/apps-engine/definition/settings';
 import type { IUIActionButton } from '@rocket.chat/apps-engine/definition/ui';
 import type { AppScreenshot, App, FeaturedAppsSection } from '@rocket.chat/core-typings';
+import type { ILogItem } from '@rocket.chat/core-typings/src/ILogs';
```

Comment:
You should add the `ILogs.ts` type file on the index to get the types from the root
```suggestion
import type { ILogItem } from '@rocket.chat/core-typings';
```